### PR TITLE
Alternative - install Windows after Clear Linux

### PR DESCRIPTION
**Pull request by bktan8**
Signed-off-by: Bun K Tan <bun.k.tan@intel.com>

### CODE REVIEW

File: source/tutorials/multi-boot/dual-boot-win.rst [hunk 1]
```diff
@@ -237,10 +246,135 @@ If you prefer not to use your BIOS to load the :guilabel:`Boot Menu` and select
...
+      :scale: 85%
+      :alt: Windows - Create new partition
+
+      Figure 7: Windows - Create new partition
+
+  .. note::
+     
+     Normally, Windows creates its own 100 MB EFI partition if none exists. 
+     In our case, where an EFI partition was created by |CL|, Windows will 
+     share use the previously-created partition.
```
Comment (tpleavitt) [inline source/tutorials/multi-boot/dual-boot-win.rst]: remove the word share or reword the meaning

File: source/tutorials/multi-boot/dual-boot-win.rst [hunk 2]
```diff
@@ -28,6 +28,8 @@ For this method, we shrink the Windows 10 OS partition to make space for |CL|.
 
 #. Right-click the primary Windows partition and select :guilabel:`Shrink Volume...`.
 
+   .. .. rst-class:: dropshadow
```
Comment (intelkevinputnam) [inline source/tutorials/multi-boot/dual-boot-win.rst]: got an extra ".." on line 31

File: source/tutorials/multi-boot/dual-boot-win.rst [hunk 3]
```diff
@@ -237,10 +247,135 @@ If you prefer not to use your BIOS to load the :guilabel:`Boot Menu` and select
...
+
+   .. rst-class:: dropshadow
+
+   .. figure:: ../../_figures/multi-boot/dual-boot-win-07.png
+      :scale: 85%
+      :alt: Windows - Create new partition
+
+      Figure 7: Windows - Create new partition
+
+  .. note::
```
Comment (mvincerx) [inline source/tutorials/multi-boot/dual-boot-win.rst]: Minor stuff. The note here needs to be left-aligned exactly with the `.. figure::` that precedes it. 
That's whats causing the numbering to restart, and showing the grey areas on GH render.

File: source/tutorials/multi-boot/dual-boot-win.rst [hunk 4]
```diff
@@ -237,10 +247,135 @@ If you prefer not to use your BIOS to load the :guilabel:`Boot Menu` and select
...
+#. Finish the Windows out-of-box-experience process.
+
+#. Restore `systemd-boot` - the bootloader |CL| uses - and add Windows to 
+   its boot menu.
+
+   At this point, you cannot boot |CL| because Windows became the default 
+   bootloader after it was installed.  Follow these steps to restore 
+   `systemd-boot`.
+
+   a. Boot up the |CL| installer.
```
Comment (mvincerx) [inline source/tutorials/multi-boot/dual-boot-win.rst]: Make this a hyperlink, please (some users may not be familiar with our docs): 

Boot up the |CL|  :ref: `installer image<install-on-target-start>`
Note: Start tickmark after colon in :ref: . Add final tickmark after `>`

Please make the exact same change on Ln 153, above under Advanced systemd option too.

**Comment by mvincerx** [comment]
LGTM. Thanks for the edit to "live image installer from a USB".  Helps unfamiliar readers.

**Pull request merged**